### ISSUE TITLE
Temperature parsing not working

### ISSUE THREAD

**Issue by mwes** [SD2E/experimental-intent-parser#117]
In GitLab by @mweston on Sep 30, 2019, 15:46

For a table entry with header "temperature" and value "37 celsius" the intent parser generates a structured request with an entry:

"temperature": ["37 celsius"],

This actually needs to be an array with units, like timepoint:

Correct example from a manually authored structured request:
https://gitlab.sd2e.org/sd2program/cp-request/blob/master/input/structured_requests/nand_2_0_omics.json#L75

```
"temperatures": [{
    "value": 37.0,
    "unit": "celsius"
}],
```

In summary, the changes should be:
1) Field name of temperature -> temperatures
2) parse celsius, Celsius, and C -> celsius unit, like with timepoints
3) Allow for an array of elements like you do for timepoints, e.g. "30, 37 C"

This also fails validation (the schema expects "temperatures", along with the other changes required here).

Example request document with a table that reproduces this: https://docs.google.com/document/d/1RenmUdhsXMgk4OUWReI2oS6iF5R5rfWU5t7vJ0NZOHw/edit#

**Comment by mwes** [comment]
In GitLab by @mweston on Sep 30, 2019, 16:28

changed the description

**Comment by mwes** [comment]
In GitLab by @nwalczak on Oct 1, 2019, 08:26

created branch [`116-temperature-parsing-not-working`](https://gitlab.sd2e.org/sd2program/experimental-intent-parser/compare/develop...116-temperature-parsing-not-working)

**Comment by mwes** [comment]
In GitLab by @nwalczak on Oct 1, 2019, 08:26

mentioned in merge request !63

**Comment by mwes** [comment]
In GitLab by @nwalczak on Oct 1, 2019, 11:42

closed via merge request !63

**Comment by mwes** [comment]
In GitLab by @nwalczak on Oct 1, 2019, 11:42

closed via commit dbc129b6deba09f25e01966a165a8b5c602bac2f

**Comment by mwes** [comment]
In GitLab by @nwalczak on Oct 1, 2019, 15:29

mentioned in commit b9734eee9d0d29c6e55c5c497f3b12c82fffa430

**Comment by mwes** [comment]
In GitLab by @nwalczak on Oct 11, 2019, 14:12

mentioned in commit dbc129b6deba09f25e01966a165a8b5c602bac2f
